### ISSUE TITLE
set up and run more than one instance on the same machine

### ISSUE THREAD

**Issue by whisere** [huridocs/uwazi#2004]
Hello. The document mentioned about changing the setting by doing below:

"However, if you require different database names, elastic indexes, etc. you can override those defaults by exporting one or more of the following environment variables:

$ export DBHOST=localhost
$ export DATABASE_NAME=uwazi_development
$ export ELASTICSEARCH_URL=http://localhost:9200
$ export INDEX_NAME=uwazi_development
$ export API_URL=/api/
$ export PORT=3000

Again, please notice that there is no need to export any value for a normal installation and only do so if you are certain you need different defaults. If these values are not correctly overridden, Uwazi will fail to run properly."

But if we need to run more than one instance on the same machine, where should we change these settings for each instance? So they use different ports and database/index names. Thanks!

**Comment by txau** [comment]
@whisere I added some extra instructions and an example of linux service configuration here:

https://github.com/huridocs/uwazi/wiki/Install-Uwazi-on-your-server

**Comment by whisere** [comment]
@txau Thank you very much! For running as linux service will require running it as root? For security I am thinking setting up different not root users for each instance and change the environment variables under the users' profiles and run separated instances including separated elasticsearch instances, will that work too? Thanks!

**Comment by whisere** [comment]
@txau Yes was thinking of creating two not root users, thanks again for the suggestions!
